api/client/bundle/client.go

```diff
@@ -24,34 +24,6 @@ func NewClient(st base.APICallCloser) *Client {
 		facade:       backend}
 }
 
-// GetChanges returns back the changes for a given bundle that need to be
-// applied.
-// GetChanges is superseded by GetChangesMapArgs, use that where possible, by
-// detecting the BestAPIVersion to use.
-func (c *Client) GetChanges(bundleURL, bundleDataYAML string) (params.BundleChangesResults, error) {
-	var result params.BundleChangesResults
-	if err := c.facade.FacadeCall("GetChanges", params.BundleChangesParams{
-		BundleURL:      bundleURL,
-		BundleDataYAML: bundleDataYAML,
-	}, &result); err != nil {
-		return result, errors.Trace(err)
-	}
-	return result, nil
-}
-
-// GetChangesMapArgs returns back the changes for a given bundle that need to be
-// applied, with the args of a method as a map.
-func (c *Client) GetChangesMapArgs(bundleURL, bundleDataYAML string) (params.BundleChangesMapArgsResults, error) {
```

Comment:
Don't remove this from the API. We're making many changes around the API to allow for multiple non juju clients to use it in the near future and it may be useful. Add a TODO to evaluate it.